### PR TITLE
improve: remove unused packages

### PR DESCRIPTION
**Pull request by james-a-morris**
WIP

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "prettier": "2.2.1",
     "prettier-plugin-solidity": "1.0.0-beta.10",
     "pretty-quick": "^2.0.1",
-    "secp256k1": "^3.7.1",
+    "secp256k1": "^5.0.1",
```
Comment (nicholaspai) [inline package.json:57]: for posterity, bumping this because of https://github.com/advisories/GHSA-584q-6j8j-r5pm

**Review by nicholaspai — APPROVED** [review]

**Review by mrice32 — APPROVED** [review]
LGTM!

**Review by daywiss — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -54,13 +54,21 @@
...
     "web3": "^1.6.0"
   },
   "resolutions": {
-    "sse4_crc32": "npm:@node-rs/crc32@1.0.0"
+    "sse4_crc32": "npm:@node-rs/crc32@1.0.0",
+    "**/ganache/secp256k1": "4.0.4",
+    "**/eccrypto/secp256k1": "3.8.1",
+    "**/eth-crypto/secp256k1": "5.0.1"
+  },
+  "overrides": {
```
Comment (nicholaspai) [inline package.json:68]: do `resolutions` and `overrides` allow us to overcome the dependencies from uma/common?

Comment (james-a-morris) [reply package.json:68]: it removes the secp256k1 issue but it only allows us to redirect deps

**Comment by james-a-morris** [comment]
> LGTM! I assume you reran yarn install after adding these overrides, right?

Yes - `yarn` and `yarn install` does not produce a diff against the latest commit

**Pull request merged**